demos/object_detection_demo_ssd_async/object_detection_demo_ssd_async.hpp

```diff
@@ -43,6 +43,8 @@ static const char raw_output_message[] = "Optional. Inference results as raw val
 /// @brief message resizable input flag
 static const char input_resizable_message[] = "Optional. Enables resizable input with support of ROI crop & auto resize.";
 
+/// @brief Message do not show processed video
+static const char no_show_processed_video[] = "Optional. Do not show processed video.";
```

Comment:
Shouldn't this be called `no_show_message`, for consistency?

Reply:
I agree, that `no_show_message` would be clearer, but `text_detection_demo` only names it as `no_show_message`, and most demos use `no_show_processed_video`.

Reply:
so, variable name is no_show_processed_video even if sample do process single image? Or in possible future, input of sample might not be image or video at all. It seems make sense to change name of variable consistently and input data type agnostic for all demos. Might be make this as a separate PR?

Reply:
Submitted the issue, that generalizes this comment: https://github.com/opencv/open_model_zoo/issues/391